app_dart/test/model/ci_yaml/ci_yaml_test.dart

```diff
@@ -176,6 +176,35 @@ void main() {
         );
       });
 
+      test('filter release_build targets from release candidate branches', () {
+        final CiYaml releaseYaml = CiYaml(
+          slug: Config.flutterSlug,
+          branch: 'flutter-2.4-candidate.3',
```

Comment:
Can you add a test for a branch that dosen't contain `flutter`, also one for `flutter-3.10-candidate.1` this way once we delete that part of the code we have also a test to change/delete

Reply:
Updated!